translator.js

```diff
@@ -5,31 +5,31 @@ const fs = require.main.require("fs");
 const LANGUAGE_DIR = __dirname + '/public/languages/';
 
 let languages, texts = null;
-loadTexts();
+
+function replaceParams(message, params) {
+    if (!params) return message;
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/exo-do/nodebb-plugin-reputation-rules/pullRequest?prid=8075881)
```suggestion
    if (!params) {return message;}
```